executor_test.go error

Following https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/build.md, the command `cd sql; go test -v ./...` complains:

```
=== RUN   TestExecutorTrainAndPredictDNN
--- FAIL: TestExecutorTrainAndPredictDNN (4.66s)
    executor_test.go:46: 
        	Error Trace:	executor_test.go:46
        	            				executor_test.go:42
        	Error:      	Should be true
        	Test:       	TestExecutorTrainAndPredictDNN
    executor_test.go:51: 
        	Error Trace:	executor_test.go:51
        	            				executor_test.go:42
        	Error:      	Should be true
        	Test:       	TestExecutorTrainAndPredictDNN
=== RUN   TestExecutorTrainAndPredictBoostedTree
--- FAIL: TestExecutorTrainAndPredictBoostedTree (5.68s)
    executor_test.go:58: 
        	Error Trace:	executor_test.go:58
        	            				executor_test.go:57
        	Error:      	Should be true
        	Test:       	TestExecutorTrainAndPredictBoostedTree
    executor_test.go:59: 
        	Error Trace:	executor_test.go:59
        	            				executor_test.go:57
        	Error:      	Should be true
        	Test:       	TestExecutorTrainAndPredictBoostedTree
```

--- a/cmd/demo/README.md
+++ b/cmd/demo/README.md
@@ -42,7 +42,7 @@ WITH
   hidden_units = [10, 20]
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_dnn_model;
+INTO sqlflow_models.my_dnn_model;
 -----------------------------
 2018/12/16 15:03:54 tensorflowCmd: run in Docker container
 Job success
@@ -54,7 +54,7 @@ Job success
 sqlflow> SELECT *
 FROM iris.test
 PREDICT iris.predict.class
-USING my_dnn_model;
+USING sqlflow_models.my_dnn_model;
 -----------------------------
 2018/12/16 15:05:58 tensorflowCmd: run in Docker container
 Job success

--- a/cmd/demo/slides/201812/automl.sql
+++ b/cmd/demo/slides/201812/automl.sql
@@ -3,5 +3,5 @@ FROM iris.train
 TRAIN Classifier
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_dnn_model;
+INTO sqlflow_models.my_dnn_model;
 

--- a/cmd/demo/slides/201812/pred.sql
+++ b/cmd/demo/slides/201812/pred.sql
@@ -1,7 +1,7 @@
 sqlflow> SELECT *
 FROM iris.test
 PREDICT iris.predict.class
-USING my_dnn_model;
+USING sqlflow_models.my_dnn_model;
 -----------------------------
 2018/12/16 15:05:58 tensorflowCmd: run in Docker container
 Job success

--- a/cmd/demo/slides/201812/preprocess.sql
+++ b/cmd/demo/slides/201812/preprocess.sql
@@ -6,5 +6,5 @@ WITH
   hidden_units = [10, 20]
 COLUMN sepal_length, sepal_width, bucket(petal_length, [0., 1., 2.]), norm(petal_width)
 LABEL class
-INTO my_dnn_model;
+INTO sqlflow_models.my_dnn_model;
 

--- a/cmd/demo/slides/201812/train.sql
+++ b/cmd/demo/slides/201812/train.sql
@@ -6,7 +6,7 @@ WITH
   hidden_units = [10, 20]
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_dnn_model;
+INTO sqlflow_models.my_dnn_model;
 -----------------------------
 2018/12/16 15:03:54 tensorflowCmd: run in Docker container
 Job success

--- a/doc/quickstart.md
+++ b/doc/quickstart.md
@@ -72,7 +72,7 @@ TRAIN DNNClassifier
 WITH n_classes = 3, hidden_units = [10, 20]
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_dnn_model;
+INTO sqlflow_models.my_dnn_model;
 -----------------------------
 ...
 Training set accuracy: 0.96721
@@ -83,7 +83,7 @@ Done training
 sqlflow> SELECT *
 FROM iris.test
 predict iris.predict.class
-USING my_dnn_model;
+USING sqlflow_models.my_dnn_model;
 ```
 - Checkout prediction result
 ```sql

--- a/doc/submitter.md
+++ b/doc/submitter.md
@@ -24,7 +24,7 @@ type ColumnType struct {
 //   hidden_units = [10, 20]
 // COLUMN sepal_length, sepal_width, petal_length, petal_width
 // LABEL class
-// INTO my_dnn_model;
+// INTO sqlflow_models.my_dnn_model;
 type TrainDescription struct {
     StandardSelect string       // e.g. SELECT * FROM iris.train
     Estimator      string       // e.g. DNNClassifier
@@ -37,7 +37,7 @@ type TrainDescription struct {
 // SELECT *
 // FROM iris.test
 // PREDICT iris.predict.class
-// USING my_dnn_model;
+// USING sqlflow_models.my_dnn_model;
 type PredDescription struct {
     StandardSelect string // e.g. SELECT * FROM iris.test
     TableName      string // e.g. iris.predict

--- a/doc/syntax.md
+++ b/doc/syntax.md
@@ -108,7 +108,7 @@ WITH layers=[100, 200],   /* a parameter of the Estimator class constructor */
 COLUMN *,                 /* all columns as raw features */
        cross(v1, v9, v28) /* plus a derived (crossed) column */
 LABEL class
-INTO my_model_table;      /* saves trained model parameters and features into a table */
+INTO sqlflow_models.my_model_table;      /* saves trained model parameters and features into a table */
 ```
 
 We see the redundancy of `*` in two clauses: `SELECT` and `COLUMN`.  The following alternative can avoid the redundancy, but cannot specify the label.
@@ -132,7 +132,7 @@ Similarly, to infer the class (fraud or regular), we could
 ```sql
 SELECT * FROM kaggle_credit_fraud_development_data
 PREDICT kaggle_credit_fraud_development_data.class
-USING my_model_table;
+USING sqlflow_models.my_model_table;
 ```
 
 ## System Architecture

--- a/doc/walkthrough.md
+++ b/doc/walkthrough.md
@@ -8,13 +8,13 @@ The following SQL statements train a TensorFlow model named `DNNClassifier`, whi
 
 ```sql
 
-SELECT * FROM a_table TRAIN DNNClassifier WITH learning_rate=0.01 INTO my_model;
+SELECT * FROM a_table TRAIN DNNClassifier WITH learning_rate=0.01 INTO sqlflow_models.my_model;
 ```
 
 And the following statement uses the trained model for prediction.
 
 ```sql
-SELECT * FROM b_table PREDICT b_table.predicted_label USING my_model;
+SELECT * FROM b_table PREDICT b_table.predicted_label USING sqlflow_models.my_model;
 ```
 
 Please be aware that the part in the above statements before the extended keyword TRAIN and PREDICT is a standard SQL statement. This feature simplifies the implementation of the SQLFlow system.

--- a/example/datasets/README.md
+++ b/example/datasets/README.md
@@ -44,6 +44,12 @@ To popularize the Irises dataset into `iris`:
 cat /popularize_iris.sql | mysql -uroot -proot
 ```
 
+To prepare database for storing machine learning models:
+
+```bash
+echo "CREATE DATABASE IF NOT EXISTS sqlflow_models;" | mysql -uroot -proot
+```
+
 ## Query
 
 In the container, run

--- a/example/jupyter/example.ipynb
+++ b/example/jupyter/example.ipynb
@@ -46,7 +46,7 @@
     "  hidden_units = [10, 20]\n",
     "COLUMN sepal_length, sepal_width, petal_length, petal_width\n",
     "LABEL class\n",
-    "INTO my_dnn_model;"
+    "INTO sqlflow_models.my_dnn_model;"
    ]
   },
   {
@@ -59,7 +59,7 @@
     "SELECT *\n",
     "FROM iris.test\n",
     "predict iris.predict.class\n",
-    "USING my_dnn_model;"
+    "USING sqlflow_models.my_dnn_model;"
    ]
   },
   {

--- a/sql/codegen_test.go
+++ b/sql/codegen_test.go
@@ -20,14 +20,14 @@ WITH
   hidden_units = [10, 20]
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_dnn_model
+INTO sqlflow_models.my_dnn_model
 ;
 `
 	testPredictSelectIris = `
 SELECT *
 FROM iris.test
 predict iris.predict.class
-USING my_dnn_model;
+USING sqlflow_models.my_dnn_model;
 `
 )
 

--- a/sql/db_test.go
+++ b/sql/db_test.go
@@ -42,6 +42,8 @@ func TestMain(m *testing.M) {
 		assertNoErr(e)
 		_, e = testDB.Exec("CREATE DATABASE IF NOT EXISTS churn;")
 		assertNoErr(e)
+		_, e = testDB.Exec("CREATE DATABASE IF NOT EXISTS sqlflow_models;")
+		assertNoErr(e)
 		defer testDB.Close()
 	default:
 		e = fmt.Errorf("unrecognized environment variable SQLFLOW_TEST_DB %s", dbms)

--- a/sql/executor_test.go
+++ b/sql/executor_test.go
@@ -16,14 +16,14 @@ WITH
   n_batches_per_layer = 20
 COLUMN sepal_length, sepal_width, petal_length, petal_width
 LABEL class
-INTO my_boosted_tree_model;
+INTO sqlflow_models.my_boosted_tree_model;
 `
 	testPredBoostedTreeOnIris = `
 SELECT *
 FROM iris.test
 WHERE class = 0 OR class = 1
 predict iris.predict.class
-USING my_boosted_tree_model;
+USING sqlflow_models.my_boosted_tree_model;
 `
 )
 

--- a/sql/model.go
+++ b/sql/model.go
@@ -19,10 +19,9 @@ type model struct {
 // SQLFlow working directory, which contains the TensorFlow working
 // directory and the trained TenosrFlow model.
 func (m *model) save(db *DB, table string) (e error) {
-	sqlfn := table
-	sqlf, e := sqlfs.Create(db.DB, sqlfn)
+	sqlf, e := sqlfs.Create(db.DB, table)
 	if e != nil {
-		return fmt.Errorf("Cannot create sqlfs file %s: %v", sqlfn, e)
+		return fmt.Errorf("Cannot create sqlfs file %s: %v", table, e)
 	}
 	defer sqlf.Close()
 
@@ -45,10 +44,9 @@ func (m *model) save(db *DB, table string) (e error) {
 // statement, and untar the SQLFlow working directory, which contains
 // the TenosrFlow model, into directory cwd.
 func load(db *DB, table, cwd string) (m *model, e error) {
-	sqlfn := table
-	sqlf, e := sqlfs.Open(db.DB, sqlfn)
+	sqlf, e := sqlfs.Open(db.DB, table)
 	if e != nil {
-		return nil, fmt.Errorf("Cannot open sqlfs file %s: %v", sqlfn, e)
+		return nil, fmt.Errorf("Cannot open sqlfs file %s: %v", table, e)
 	}
 	defer sqlf.Close()
 

--- a/sql/model.go
+++ b/sql/model.go
@@ -19,7 +19,7 @@ type model struct {
 // SQLFlow working directory, which contains the TensorFlow working
 // directory and the trained TenosrFlow model.
 func (m *model) save(db *DB, table string) (e error) {
-	sqlfn := fmt.Sprintf("sqlflow_models.%s", table)
+	sqlfn := table
 	sqlf, e := sqlfs.Create(db.DB, sqlfn)
 	if e != nil {
 		return fmt.Errorf("Cannot create sqlfs file %s: %v", sqlfn, e)
@@ -45,7 +45,7 @@ func (m *model) save(db *DB, table string) (e error) {
 // statement, and untar the SQLFlow working directory, which contains
 // the TenosrFlow model, into directory cwd.
 func load(db *DB, table, cwd string) (m *model, e error) {
-	sqlfn := fmt.Sprintf("sqlflow_models.%s", table)
+	sqlfn := table
 	sqlf, e := sqlfs.Open(db.DB, sqlfn)
 	if e != nil {
 		return nil, fmt.Errorf("Cannot open sqlfs file %s: %v", sqlfn, e)

--- a/sql/parser_test.go
+++ b/sql/parser_test.go
@@ -25,12 +25,10 @@ COLUMN
   bucketize(last_name, 1000),
   cross(embedding(emplyoee.name), bucketize(last_name, 1000))
 LABEL employee.salary
-INTO
-  my_dnn_model
-;
+INTO sqlflow_models.my_dnn_model;
 `
 	testPredictSelect = testStandardSelectStmt + `PREDICT db.table.field
-USING my_dnn_model;`
+USING sqlflow_models.my_dnn_model;`
 )
 
 func TestStandardSelect(t *testing.T) {

--- a/sql/parser_test.go
+++ b/sql/parser_test.go
@@ -65,7 +65,7 @@ func TestTrainParser(t *testing.T) {
 		`cross(embedding(emplyoee.name), bucketize(last_name, 1000))`,
 		r.columns[2].String())
 	a.Equal("employee.salary", r.label)
-	a.Equal("my_dnn_model", r.save)
+	a.Equal("sqlflow_models.my_dnn_model", r.save)
 }
 
 func TestPredictParser(t *testing.T) {
@@ -74,7 +74,7 @@ func TestPredictParser(t *testing.T) {
 	a.NoError(e)
 	a.True(r.extended)
 	a.False(r.train)
-	a.Equal("my_dnn_model", r.model)
+	a.Equal("sqlflow_models.my_dnn_model", r.model)
 	a.Equal("db.table.field", r.into)
 }
 

--- a/sql/verifier_test.go
+++ b/sql/verifier_test.go
@@ -85,20 +85,20 @@ WITH
   hidden_units = [10, 20]
 COLUMN gender, tenure, TotalCharges
 LABEL class
-INTO my_dnn_model;`)
+INTO sqlflow_models.my_dnn_model;`)
 	a.NoError(e)
 
 	predParse, e := newParser().Parse(`SELECT gender, tenure, TotalCharges
 FROM churn.train LIMIT 10
 PREDICT iris.predict.class
-USING my_dnn_model;`)
+USING sqlflow_models.my_dnn_model;`)
 	a.NoError(e)
 	a.NoError(verifyColumnNameAndType(trainParse, predParse, testDB))
 
 	predParse, e = newParser().Parse(`SELECT gender, tenure
 FROM churn.train LIMIT 10
 PREDICT iris.predict.class
-USING my_dnn_model;`)
+USING sqlflow_models.my_dnn_model;`)
 	a.NoError(e)
 	a.EqualError(verifyColumnNameAndType(trainParse, predParse, testDB),
 		"predFields doesn't contain column TotalCharges")

